testpmd-lb-operator/roles/loadbalancer/templates/deployment.yml

```diff
@@ -42,6 +42,9 @@ spec:
                 - pkt-gen
             topologyKey: kubernetes.io/hostname
       serviceAccountName: loadbalancer-account
+{% if high_perf_runtime is defined and high_perf_runtime|length %}
```

Comment:
high_perf_runtime is not used in LoadBalancer CR, so I think this line is not really needed here, we just need this for the cnfapp, which is under testpmd-operator

Reply:
I miss the 'irq-load-balancing' annotation, we want this on the testpmd-lb